doc-content/enterprise-only/weblogic/wls-configure-embedded-engine-proc.adoc

```diff
@@ -0,0 +1,286 @@
+[id='wls-configure-embedded-engine-proc']
+ifdef::PAM[]
+= Configuring an embedded {PROCESS_ENGINE} and {DECISION_ENGINE} in {WEBLOGIC}.
+endif::[]
+
+ifdef::DM[]
+= Configuring an embedded {DECISION_ENGINE} in {WEBLOGIC}
+endif::[]
+
+An embedded engine is a light-weight work-flow engines and rule engine which executes your decisions and business processes. They can be embedded as part of your application or deployed as a service possibly in the cloud.
```

Comment:
An embedded engine is a light-weight work-flow and rule engine...
An embedded engine can be part of your {PRODUCT} appliction or they can be deployed as a service, for example in the cloud. Do you mean on OpenShift, or in the cloud generally?

Reply:
In the cloud, not OpenShift.